README.rst

```diff
@@ -35,6 +37,18 @@ following packages (or similar, depending on your operating system) installed:
 - ncurses-dev
 - openssl-dev
 
+syncstorage-rs uses the following additional dependencies:
+
+- cmake
+- golang
+- libcurl4-openssl-dev
+- libssl-dev
+- pkg-config
+- Rust stable
```

Comment:
I wonder if we need to add a link to `rustup.rs` here? Not sure if `rustlang` is available on all the platforms yet. Did a quick search on [Debian buster](https://packages.debian.org/buster/devel/) and could only find the source.